src/config/wagmi.ts

```diff
@@ -30,7 +30,7 @@ const getWallets = () => {
 export const { chains: availableChains, publicClient } = (() => {
   if (TEST_MODE) {
     return configureChains(
-      [sepolia],
+      [sepolia, optimismSepolia],
```

Comment:
this is the config for e2e tests, so no changes are needed here
```suggestion
      [sepolia],
```